docs/sources/reference-http-api/index.md

```diff
@@ -1,17 +0,0 @@
----
-aliases:
```

Comment:
should create an http reference when we are confident in our api i guess

Reply:
Yes that is covered in #2156